Meet The Teacher/manifest.php

```diff
@@ -25,8 +25,8 @@
 $entryURL = 'meetTheTeacher_view.php';
 $type = 'Additional';
 $category = 'Other';
-$version = '1.1.06';
-$author = 'Jim Speirs, Sandra Kuipers & Ross Parker';
+$version = '1.1.04';
```

Comment:
Should be 1.1.07, as this is for new installs, and so version.php is not used.

Reply:
It should, good catch